src/components/VideoThumbnail/README.md

```diff
@@ -0,0 +1,106 @@
+---
+name: Video thumbnail
+category: Images and icons
+keywords:
+  - video
+  - VideoThumbnail
+  - updates
+  - new features
+  - video thumbnail
+  - feature thumbnail
+  - education
+  - contextual learning system
+---
+
+# Video thumbnail
+
+Video thumbnails provide consistent controls for video cards in the contextual learning system. When clicked, conditionally present a video player in the format of your choice. For example, within a modal or a full screen container.
+
+---
+
+## Best practices
+
+- Use a thumbnail that helps to communicate the subject of the video or merchant benefit
+
+---
+
+## Content guidelines
+
+### Thumbnail image
+
+Thumbnail image should be:
+
+- Relevant: should describe the video through the use of images or be an image of a section in the video
+- Follow the 16:9 ratio, otherwise the image may appear cropped
+- Avoid cropping the person’s head if the thumbnail shows a person
+
+<!-- usagelist -->
+
+#### Do
+
+Have an image with a non-cropped head, clean aesthetic that matches shopify brand and is descriptive
+of content
+
+#### Don’t
+
+Use a random, unintentional, blurry screenshot from the video
+
+#### Do
+
+Include a video timestamp
+
+#### Don’t
+
+Omit the timestamp
+
+<!-- end -->
+
+---
```

Comment:
These guidelines are more fitting as best practices since the component doesn't comprise any content aside from the `accessibilityLabel` for the play button, which is covered in the `Accessibility` section.

```suggestion
```